tsl/src/continuous_aggs/refresh.h

```diff
@@ -8,7 +8,9 @@
 
 #include <postgres.h>
 #include <fmgr.h>
+#include "continuous_aggs/materialize.h"
```

Comment:
Why is this include needed?

`ContinuousAgg` is defined in `src/continuous_agg.h`, but I would also consider a pre-declaration:

```
typedef struct ContinuousAgg ContinuousAgg;
```